front/src/modules/ui/components/menu/DropdownMenuItem.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+
+export const DropdownMenuItem = styled.div`
+  --horizontal-padding: 6px;
```

Comment:
this looks suspcious to me. Can we break them into what these 6px come from? (maybe 4px from spacing + 2 px of borders?)